diff --git a/node_modules/@tiptap/core/dist/tiptap-core.esm.js b/node_modules/@tiptap/core/dist/tiptap-core.esm.js
index e825d63..ecc9e27 100644
--- a/node_modules/@tiptap/core/dist/tiptap-core.esm.js
+++ b/node_modules/@tiptap/core/dist/tiptap-core.esm.js
@@ -2798,7 +2798,7 @@ function isExtensionRulesEnabled(extension, enabled) {
 
 function findDuplicates(items) {
     const filtered = items.filter((el, index) => items.indexOf(el) !== index);
-    return [...new Set(filtered)];
+    return Array.from(new Set(filtered));
 }
 
 class ExtensionManager {
